### PR TITLE
Main

### PR DESCRIPTION
**Pull request by Joel-Cripps**
Change master branch to main. Will set to default once this approved. 

Changed values where we can. Added comments to others.

### CODE REVIEW

**Review by JaydenLiang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
 # FortiGate Autoscale
+
 A collection of **Node.js** modules and cloud-specific templates which support autoscale functionality for groups of FortiGate-VM instances on various cloud platforms.
```
Comment (klc178) [inline README.md]:
```suggestion
A collection of **Node.js** modules and cloud-specific templates that support autoscale functionality for groups of FortiGate-VM instances on various cloud platforms.
```

File: README.md [hunk 2]
```diff
@@ -1,38 +1,42 @@
...
 
 This project contains the code and templates for the **FortiGate Autoscale for GCP** deployment.
 For autoscale on **AliCloud** see the [alicloud-autoscale](https://github.com/fortinet/alicloud-autoscale/) repository.
 For autoscale on **Amazon AWS** and **Microsoft Azure** see the [fortigate-autoscale](https://github.com/fortinet/fortigate-autoscale/) repository.
 
 ## Supported Platforms
+
 This project supports autoscale for the cloud platforms listed below.
```
Comment (klc178) [inline README.md]:
```suggestion
This project supports autoscale for the cloud platform listed below.
```

**Review by klc178 — CHANGES_REQUESTED** [review]

File: main.tf [hunk 3]
```diff
@@ -276,10 +276,12 @@ resource "google_cloudfunctions_function" "function" {
       RESOURCE_TAG_PREFIX        = "${var.cluster_name}"
       PAYG_SCALING_GROUP_NAME    = "${var.cluster_name}-${random_string.random_name_post.result}",
       BYOL_SCALING_GROUP_NAME    = "${var.cluster_name}-${random_string.random_name_post.result}",
+      #TODO: change value toMAIN once core logic is changed.
       MASTER_SCALING_GROUP_NAME  = "${var.cluster_name}-${random_string.random_name_post.result}",
       HEART_BEAT_LOSS_COUNT      = "${var.HEART_BEAT_LOSS_COUNT}",
       SCRIPT_TIMEOUT             = "${var.SCRIPT_TIMEOUT}"
-      MASTER_ELECTION_TIMEOUT    = "${var.MASTER_ELECTION_TIMEOUT}",
+       #TODO: change value toMAIN once core logic is changed.
```
Comment (jamie-pate) [inline main.tf]: to MAIN

also, **MAIN** or **PRIMARY**

File: vars.tf [hunk 4]
```diff
@@ -112,16 +112,14 @@ variable "target_size" {
...
 #### Optional Configurations ####
 #Cloud Function timeout. In seconds
 variable "SCRIPT_TIMEOUT" {
   type    = number
   default = 500
 }
-# Time before a master election times out. In seconds
-variable "MASTER_ELECTION_TIMEOUT" {
+# Time before an election times out. In seconds
+variable "PRIMARY_ELECTION_TIMEOUT" {
```
Comment (jamie-pate) [inline vars.tf:122]: **PRIMARY** election or **MAIN**

**Review by jamie-pate — CHANGES_REQUESTED** [review]

**Comment by Joel-Cripps** [comment]
Added review changes,
Take a look and let me know

**Review by klc178 — APPROVED** [review]

File: vars.tf (hunk 4 above)
Comment (Joel-Cripps) [reply vars.tf:122]: erased the main comment

File: main.tf (hunk 3 above)
Comment (Joel-Cripps) [reply main.tf]: erased the main comment

**Review by jamie-pate — APPROVED** [review]

**Pull request merged**